src/OSPSuite.SimModelNative/src/Species.cpp

```diff
@@ -82,6 +82,7 @@ void Species::LoadFromXMLNode (const XMLNode & pNode)
 	if ((_containerPath.find("SalivaGland") != string::npos) ||
 		(_containerPath.find("IgG_Source") != string::npos) ||
 		(_containerPath.find("ParticleBin_") != string::npos)||
+		(_containerPath.find("|Events|") != string::npos) ||
 		(_containerPath.find("|Applications|") != string::npos))
```

Comment:
Ignoring all molecule amounts under the **Events** node when checking for negative values
(also keeping **Applications** in the exclusions list in order to run the simulations created with <V12)

Reply:
crazy